test/test.js

```diff
@@ -42,6 +42,7 @@
   var phantom = root.phantom,
       amd = root.define && define.amd,
       argv = root.process && process.argv,
+      jsArrayFrom = Array.from,
```

Comment:
`_.toArray` conforms to it so that's fine and lets it be tested in Chrome and Node which lacks `Array.from`.